### PR TITLE
As of now we're just doing an abstract definition of events

### PR DESCRIPTION
**Pull request by duglin**
See: https://github.com/cloudevents/spec/pull/64#issuecomment-364993279
for the full context

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by duglin** [comment]
#64 was merged and this PR is meant to go along with it - so please review and hopefully we can merge it on the next call.

**Comment by ultrasaurus** [comment]
I thought the working group planned to define what was needed to make events interoperable.  Just that we wanted to define the attributes and their meaning first before worrying about encoding them (although at one point we talked about a parallel track).

will work on https://github.com/cloudevents/spec/issues/74 to clarify

File: spec.md [hunk 1]
```diff
@@ -50,6 +50,10 @@ At this time the specification is focused on the following scope:
   with an initial focus on HTTP and Functions-as-a-Service/Serverless computing.
 * Determine process and overall governance of the specification.
 * Discuss additional architecture components that complement this specification.
+* As of now this specification defines the abstract definition of an Event.
+  Whether or not this specification, or working group, will define what this
+  definition looks like as an "on the wire" representation or how it is
+  presented to an application receiving the event is still to-be-determined.
```
[the remark this answers is not in the document]
Comment (duglin) [reply spec.md]: TBD - not out of scope. This is just a way of saying: we're starting with the abstract definition of the properties of an event first and we'll deal with the other stuff (like serialization) later. Baby steps.

**Comment by duglin** [comment]
Closing since I think #114 will be adopted and it removes this entire section - so not needed any more.

**Pull request closed without merging**